composer.json

```diff
@@ -68,6 +68,9 @@
             },
             {
                 "file": "app/config/samlstepupproviders_parameters.yml"
+            },
+            {
+                "file": "app/config/global_view_parameters.yml"
```

Comment:
Yes. This is communicated to the user/developer by showing the default value in the same representation:
```
Some parameters are missing. Please provide them.
support_url ({ en_GB: 'https://support.surfconext.nl/faq-strong-authentication', nl_NL: 'https://support.surfconext.nl/faq-sterke-authenticatie' }): 
```